### PR TITLE
Update tool string

### LINKED ISSUE #6417: Update all tool string containing 'pub get' to 'flutter packages get'
We recently changed flutter so that you need to run `flutter packages get`, not `pub get`.

However, some (many?) status messages still say that we are running `pub get` thus potentially causing confusion. For example, when running flutter for the first time:

```
$ flutter doctor
Downloading Material fonts...                        2.2s
Downloading package sky_engine...                    1.4s
Running 'pub get' in sky_engine...                   0.4s
Downloading package sky_services...                  0.4s
Running 'pub get' in sky_services...                 3.1s
Downloading package flutter_services...              0.3s
Running 'pub get' in flutter_services...             1.7s
```

### PR DESCRIPTION
**Pull request by warent**
Fixes #6417
'pub get' should be 'flutter packages get'
'pub run tests' should be 'flutter test'
'pub upgrade' should be 'flutter packages upgrade'

### CODE REVIEW

File: packages/flutter_tools/README.md [hunk 1]
```diff
@@ -4,5 +4,5 @@ Tools for building Flutter applications.
 
 To run the tests:
 
-- pub get
-- pub run test
+- flutter packages get
```
Comment (eseidelGoogle) [inline packages/flutter_tools/README.md]: I don't think these need changing for flutter_tools.  flutter_tools doesnt' depend on any flutter packages last I knew, it runs in the normal dart vm.  @danrubel would know.

File: packages/flutter_tools/lib/src/commands/create.dart [hunk 2]
```diff
@@ -17,10 +17,9 @@ import '../template.dart';
 
 class CreateCommand extends FlutterCommand {
   CreateCommand() {
-    argParser.addFlag('pub',
-      defaultsTo: true,
-      help: 'Whether to run "pub get" after the project has been created.'
-    );
+
+    usesPubOption();
```
Comment (eseidelGoogle) [inline packages/flutter_tools/lib/src/commands/create.dart]: This seems like a strange name for a function with side-effects, but it also doesn't look like you created this function.  @devoncarew

File: packages/flutter_tools/lib/src/commands/update_packages.dart [hunk 3]
```diff
@@ -18,7 +18,7 @@ class UpdatePackagesCommand extends FlutterCommand {
   UpdatePackagesCommand({ this.hidden: false }) {
     argParser.addFlag(
       'upgrade',
-      help: 'Run "pub upgrade" rather than "pub get".',
+      help: 'Run "flutter packages upgrade" rather than "flutter packages get".',
```
Comment (eseidelGoogle) [inline packages/flutter_tools/lib/src/commands/update_packages.dart]: This is describing what the command does.  It runs pub upgrade or pub get under the covers, it just happens to also set FLUTTER_ROOR (which is all `flutter packages *` does anyway).

**Review by eseidelGoogle — CHANGES_REQUESTED** [review]
I think half of these are correct.  Users of flutter should only need to think about `flutter packages *`, developers of flutter_tools are just normal dart vm developers, and developers of flutter (including help messages for `flutter packages`) may need to reference `pub get`, etc. since `flutter packages` (currently) uses pub under the covers.

**Comment by warent** [comment]
I'm seeing now something fragile happens with the --pub command in create.dart because the order should occur at an overridden place rather than the default. Sorry about that, fixing....

**Comment by eseidelGoogle** [comment]
You should feel free to update this CL, or you can close it and start a new.  Completely up to you!  There were definitely some useful parts here, just not all of it was correct.

Thanks again for your help!  We really appreciate it!

**Pull request closed without merging**